src/current/v24.3/set-up-logical-data-replication.md

```diff
@@ -0,0 +1,189 @@
+    ----------------------+---------+---------------------------+------------------
+    1012877040439033857   | running | {database.public.table}   | NULL
+    (1 row)
+    ~~~
+
+    If you're setting up bidirectional LDR streams, both clusters will have a history retention job and an LDR job running.
+
+1. Move on to [Step 4](#step-4-optional-set-up-bidirectional-ldr-streams) to set up a second LDR stream. Or, once you have set up your required LDR streams, refer to [Step 5](#step-5-monitor-the-ldr-jobs) to monitor the jobs in the DB Console.
+
+## Step 4. (Optional) Set up bidirectional LDR streams
+
+At this point, you've set up one LDR stream from cluster A as the source to cluster B as the destination. 
+
+1. To set up a stream flowing in the opposite direction, complete [Step 2](#step-2-connect-from-the-destination-to-the-source) and [Step 3](#step-3-start-the-ldr-stream) again. Cluster B will now be the source, and cluster A will be the destination.
+1. After the two streams are running, you can check the number of rows on each cluster with:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SELECT COUNT(*) from {table_name};
+    ~~~
+
+    {% comment  %} TODO: Add output here when example included.
+    To verify that data at a certain point in time is correct on the destination cluster, refer to [Data verification](). {% endcomment %}
+
+## Step 5. Monitor the LDR jobs
+
+In this step, you'll access the [DB Console]({% link {{ page.version.version }}/ui-overview.md %}) and monitor the status and metrics for the created LDR jobs. Depending on which cluster you would like to view, follow the instructions for either the source or destination.
+
+1. Access the DB Console at `http://{node IP or hostname}:8080` and enter your user's credentials.
+1. Navigate to the [**Jobs** page]({% link {{ page.version.version }}/ui-jobs-page.md %}) to view a list of all jobs. Use the job **Type** dropdown and select **Replication Producer**. On the source cluster for an LDR stream, this will display the history retention job. This will run while the LDR stream is active to protect changes to the table from [garbage collection]({% link {{ page.version.version }}/architecture/storage-layer.md %}#garbage-collection) until they have been replicated to the destination cluster.
+1. Use the job **Type** dropdown and select **Logical Replication Ingestion**. On the destination cluster for an LDR stream, this page will display the logical replication stream job. There will be a progress bar in the **Status** column when the LDR stream is replicating a table with existing data. This progress bar shows the status of the initial scan, which backfills the destination table with the existing data.
+1. Click on **Metrics** in the left-hand navigation menu. Use the **Dashboard** dropdown to select **Logical Data Replication**. This page shows graphs for monitoring the LDR stream. 
```

Comment:
Do you plan on also adding the list of metrics that you can see on DB console?

Reply:
See the comment below this line. Plan to add the details to the monitoring page. But can add a shortened form here too.

Reply:
I'll do this with the monitoring PR, I think. So, then I know what I'm talking about better. Happy to take a list though if you want to add that here in this PR.